src/store/index.js

```diff
@@ -11,19 +12,16 @@ Vue.use(Vuex);
 export default new Vuex.Store({
   strict: true,
   state: {
-    online: true,
   },
   getters: {
-    online: (state) => state.online,
   },
   mutations: {
```

Comment:
Does this need an empty mutations object?

Reply:
Nope ;-) Fix is on the way